docs/package.json

```diff
@@ -28,7 +28,7 @@
     "@docusaurus/module-type-aliases": "3.6.3",
     "@docusaurus/types": "3.6.3",
     "@tsconfig/recommended": "^1.0.8",
-    "docusaurus-plugin-typedoc": "^1.0.5",
+    "docusaurus-plugin-typedoc": "^1.1.1",
     "typedoc": ">=0.26.11",
```

Comment:
Hmmm... `typedoc` should've been bumped here I think?

Reply:
I'm noticing `typedoc-plugin-markdown` also isn't being bumped, but both of the latest versions are included in `package-lock.json` 🤔 

Hoping another "recreate" attempt in a few days might fix this! I thought the added `>=` was confusing but the dependency below kept `^` without receiving an update...

Reply:
FWIW that might be unrelated to current `build` failures related to the sidebar here:

https://github.com/slackapi/node-slack-sdk/actions/runs/12324871475/job/34403193412#step:5:10

```
../packages/socket-mode/tsconfig.json:15:14 - error TS6053: File '@tsconfig/recommended/tsconfig.json' not found.

15   "extends": "@tsconfig/recommended/tsconfig.json",
```